tests/committerparser.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
+import sys
```

Comment:
This will need a `from __future__ import absolute_path` at the top here for python 3 compatibility.